public/app/plugins/panel/canvas/CanvasPanel.tsx

```diff
@@ -131,7 +153,55 @@ export class CanvasPanel extends Component<Props, State> {
     return changed;
   }
 
+  inlineEditButtonClick = (open: boolean) => {
+    this.setState({ openInlineEdit: open });
+  };
+
+  renderInlineEdit = () => {
+    const dashboard = getDashboardSrv().getCurrent();
```

Comment:
lets avoid this if possible -- I think we have all properties we need without breaking out of the frame